libraries/platforms/bigblade-vcs/link.mk

```diff
@@ -71,14 +71,15 @@ VCS_VSOURCES = $(VHEADERS) $(VSOURCES)
 
 $(BSG_MACHINExPLATFORM_PATH)/debug/simv: VCS_VFLAGS += +plusarg_save +vcs+vcdpluson +vcs+vcdplusmemon +memcbk -debug_pp
 
-$(BSG_MACHINExPLATFORM_PATH)/saifgen/simv $(BSG_MACHINExPLATFORM_PATH)/exec/simv: VDEFINES += BSG_MACHINE_DISABLE_VCORE_PROFILING
-$(BSG_MACHINExPLATFORM_PATH)/saifgen/simv $(BSG_MACHINExPLATFORM_PATH)/exec/simv: VDEFINES += BSG_MACHINE_DISABLE_CACHE_PROFILING
-$(BSG_MACHINExPLATFORM_PATH)/saifgen/simv $(BSG_MACHINExPLATFORM_PATH)/exec/simv: VDEFINES += BSG_MACHINE_DISABLE_ROUTER_PROFILING
```

Comment:
Did the saifgen rules get removed? Or am I just being confused by git

Reply:
you're being confused by git. i added pc-histogram as one of the targets for which this rule applies and saifgen is still one of those targets.

Reply:
Not sure. Maybe? I figured I'd let this new tool breath a bit before I decided for other people how and when it should be used.

Reply:
That's reasonable to me. 

:ship: it

Reply:
But I'll happily defer to your opinion...